.github/workflows/cli.yml

```
@@ -44,7 +49,7 @@ jobs:
           mkdir -p $GOPATH/bin
           curl -L -o $GOPATH/bin/gfmrun "https://github.com/urfave/gfmrun/releases/download/v1.2.14/gfmrun-$(go env GOOS)-amd64-v1.2.14"
           chmod +x $GOPATH/bin/gfmrun
-          npm install markdown-toc
+          npm install -g markdown-toc
```

Request: We should pin this dependency version to prevent future breakages. 

```suggestion
          npm install -g markdown-toc@1.2.0
```